pkg/version/version.go

```diff
@@ -38,8 +38,8 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                           = "0.7.0-beta.1.dev.1"
-	PrevReleaseVersion                  = "0.7.0-beta.1"
+	Promscale                           = "0.7.0-beta.1.dev.2"
```

Comment:
Any specific reason behind the version change?

Reply:
because we want to run migration files for telemetry (`sql/versions/dev/0.7.0-beta.1.dev/2-telemetry`).